.github/workflows/README.md

```diff
@@ -0,0 +1,7 @@
+# GitHub workflows
+
+## Reusabe workflows
+
+This folder contains some workflows that can be re-used from other repositories:
+
+- [publish-techdocs](./publish-techdocs.md)
```

Comment:
Could we have a one-sentence summary here please? :pray:

Reply:
Done :)